src/euler3d_mappedGrid.f90

```diff
@@ -0,0 +1,679 @@
+MODULE mg
```

Comment:
This is not a huge deal but could you name the module something more illustrative?  This would help with readability mostly but we also in general name modules the same as the containing source file to simplify the build process since the `*.mod` file takes on the name of the module rather than the source file.